NotificationApi/NotificationApi.IntegrationTests/Database/Commands/CreateEmailNotificationCommandTests.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NotificationApi.DAL;
+using NotificationApi.DAL.Commands;
+using NotificationApi.Domain.Enums;
+using NUnit.Framework;
+
+namespace NotificationApi.IntegrationTests.Database.Commands
+{
+    public class CreateEmailNotificationCommandTests : DatabaseTestsBase
+    {
+        private CreateEmailNotificationCommandHandler _handler;
+        private Guid _notificationId;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var context = new NotificationsApiDbContext(NotifyBookingsDbContextOptions);
+            _handler = new CreateEmailNotificationCommandHandler(context);
+        }
+
+        [Test]
+        public async Task Should_save_new_notification()
+        {
+            var notificationType = (int)NotificationType.CreateIndividual;
+            const string email = "test@email.com";
+            var participantId = Guid.NewGuid();
+            var hearingId = Guid.NewGuid();
+            var command = new CreateEmailNotificationCommand(notificationType, email, participantId, hearingId);
+
+            await _handler.Handle(command);
+
+            command.NotificationId.Should().NotBeEmpty();
```

Comment:
no longer needed set in ctor of CreateEmailNotificationCommand
can we verify its been added to db?

Reply:
its done on next file actually but should be here too :) 
Notification updatedNotification;
            await using (var db = new NotificationsApiDbContext(NotifyBookingsDbContextOptions))
            {
                updatedNotification = await db.Notifications.SingleOrDefaultAsync(x => x.Id == notification.Id);
            }